.coderabbit.yaml

```
@@ -0,0 +1,88 @@
+knowledge_base:
+  opt_out: false
+  learnings:
+    scope: auto
+  issues:
+    scope: auto
+  jira:
+    project_keys: []
+  linear:
+    team_keys: []
+  pull_requests:
+    scope: auto
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)